### ISSUE TITLE
[Keyboard Navigation – Make Code – Import URL] 'Open project URL' pop-up window is not getting close via 'ESC' key.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3580]
**User Experience:**
User depending on Keyboard might get affected as user is unable to close the 'Dialog' using the "ESC " key which may hinder the user to use the application effectively.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to Import URL option and navigate through the all the controls of dialog.
6. Verify that 'Open project URL' pop-up window is not getting close via 'ESC' key.

**Actual Result:**
'Open project URL' pop-up window is not getting close via 'ESC' key.

**Expected Result:**
'Open project URL' pop-up window should be close via 'ESC' key.

**Note:**
1. Same issue exists on Firefox & Chrome.
2. Same issue exists under 'Import File' pop-up.
3. Same issue exists under 'Sign in with GitHub' pop-up.
4. Same issue exist under  'Pair Device' & 'Download to micro:bit pop up.

**WCAG Reference:**
[WCAG 2.4.5 - Multiple Ways
](https://www.w3.org/WAI/WCAG21/Understanding/multiple-ways.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox, Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.4.5, #Win10-Edge(Chromium)

[WCAG2.4.5_Import URL.zip](https://github.com/microsoft/pxt-microbit/files/5487000/WCAG2.4.5_Import.URL.zip)
